examples/Integration Mappings/README.md

```diff
@@ -0,0 +1,97 @@
+# Integration Mappings (Slack)
+
+This script helps manage the folder mappings between Slack and Box if using Box as the content store for Slack. It creates a list of current Slack channel and Box folder mappings and can create or update mappings based on input csv. This script will maintain all permissons. 
+
+## Setup Pre-Requisites
+1. Clone this github repo or download files from the `/examples` directory
+   ```bash
+   git clone https://github.com/box/boxcli.git
+   ```
+2. Install PowerShell or .Net core.
+   > If you encounter issues make sure you install both dotnet core and PowerShell
+	1. For MacOS & Linux, Install the latest version of [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.2).
+	2. For Windows, Install the latest version of [dotnet core](https://dotnet.microsoft.com/download).
+
+3. Test PowerShell by running the `pwsh` command in your terminal.
+    ```bash
+    pwsh
+    ```
+
+    ```
+    mikechen@mbp Integration Mappings % pwsh
+    PowerShell 7.2.1
+    Copyright (c) Microsoft Corporation.
+	
+    https://aka.ms/powershell
+    Type 'help' to get help.
+	
+    PS /Users/mikechen>
+    ```
+4. Configure and install the Box CLI using the OAuth [CLI Setup Quick Start][oauth-guide]. Make sure the user you use is an admin or co-admin.
+5. Configure and [install Box for Slack][install-slack] in the relevant Slack workspaces and orgs
+6. Box as the [content layer for Slack][content-layer] in enabled
+
+## Run the script
+
+Change the directory to the folder containing the script. In this example, it is the `Integration Mappings` folder.
+```pwsh
+rvb@lab:~/box-cli/examples/Integration Mappings$ pwsh
+PowerShell 7.2.4
+Copyright (c) Microsoft Corporation.
+
+https://aka.ms/powershell
+Type 'help' to get help.
+
+PS /home/rvb/box-cli/examples/Integration Mappings>
+```
+
+Run the script with EXTRACT to extract current mappings:
+```pwsh
+./integration-mappings.ps1 -Action EXTRACT
+```
+
+or
+
+Run the script with UPDATE to update current mappings:
```

Comment:
Can we add two .csv example files, one for update e.g. `mapping_update.csv` and the other one for create e.g. `mapping_create.csv` to show that depending on the selected `ACTION` we use a different .csv format?

`mapping_update.csv`

```
"Id","BoxItemId"
123, 9876543321
```

`mapping_create.csv`

```
"BoxItemId","ChannelId","SlackOrgId"
"9876543321","AB89X56Y","789"
```

We can also mention, that when using the `CREATE` action, we can use `SlackOrgId` or `SlackWorkspaceId` header.